docs/pages/core-concepts.mdx

```diff
@@ -38,16 +38,22 @@ Read our guides to how [authorization](./architecture/authorization.mdx) and
 
 ### Teleport Proxy Service
 
-The **Teleport Proxy Service** allows for secure access to resources in a your
-infrastructure from the public internet without the need for a VPN. 
+The **Teleport Proxy Service** allows for secure access to resources in your
+infrastructure from the public internet without the need for a VPN.  The
+Proxy Service provides connectivity for users, machines and resources
+within the cluster.
```

Comment:
I think we can condense this paragraph. The second sentence repeats the idea in the first sentence that the Proxy Service allows for secure access to resources in your infrastructure.